tests/unit/test__helpers.py

```diff
@@ -728,17 +728,17 @@ def test_w_string(self):
         self.assertEqual(self._call_fut(ZULU), ZULU)
 
     def test_w_datetime(self):
-        from google.cloud._helpers import _microseconds_from_datetime
+        from google.cloud._helpers import _RFC3339_MICROS
 
         when = datetime.datetime(2016, 12, 20, 15, 58, 27, 339328)
-        self.assertEqual(self._call_fut(when), _microseconds_from_datetime(when) / 1e6)
+        self.assertEqual(self._call_fut(when), when.strftime(_RFC3339_MICROS))
```

Comment:
Please update these tests to compare against the actual string produced. I trust that `when.strftime(_RFC3339_MICROS)` is doing the right thing, but I'd be much more comfortable seeing the actual formatted string here and in the other unit tests.